### ISSUE TITLE
Expression form uses common name when displaying organisms

### ISSUE THREAD

**Issue by jwest60** [tripal/tripal_analysis_expression#263]
organism names wont show up if the common name field isnt set

![image](https://user-images.githubusercontent.com/32902460/50016681-e01de400-ff97-11e8-9351-bdf20cd9daad.png)

refactor to `chado_get_organism_select_options`

**Comment by jwest60** [comment]
`chado_get_organism_select_options` doesn't seem to work.
```
PDOException: SQLSTATE[42P01]: Undefined table: 7 ERROR: relation "public.chado_organism" does not exist LINE 1: SELECT * FROM public.chado_organism CO ... ^: SELECT * FROM public.chado_organism CO INNER JOIN chado.organism O ON O.organism_id = CO.organism_id ORDER BY O.genus, O.species ; Array ( ) in chado_query() (line 1719 of /home/vagrant/code/tripal/sites/all/modules/tripal/tripal_chado/api/tripal_chado.query.api.inc).
```
Looks like there's something wrong with the sql query.

However, similar logic can be applied so we can change
`$organisms[$org->organism_id] = $org->common_name;`
to
`$organisms[$org->organism_id] = $org->genus . ' ' . $org->species;`

**Comment by bradfordcondon** [comment]
@jwest60 call `chado_get_organism_select_options(FALSE)`

**Comment by jwest60** [comment]
yup, that seems to have done the trick. thanks @bradfordcondon

**Comment by jwest60** [comment]
Fixed on #265
